Jenkinsfile

```diff
@@ -83,17 +83,15 @@ pipeline {
               def parallelTasksWithoutAsyncHooks = [:]
               node['NODEJS_VERSION'].each{ version ->
                 parallelTasks["Node.js-${version}"] = generateStep(version: version)
-                parallelTasksWithoutAsyncHooks["Node.js-${version}-async-hooks-false"] = generateStep(version: version)
+                if (!version.startsWith('6')) {
+                  parallelTasks["Node.js-${version}-async-hooks-false"] = generateStep(version: version, disableAsync: true)
```

Comment:
I think the name can be a bit confusing as there's also a feature in Node.js called `async`. So to make our lifes easier in the future, I suggest naming it `disableAsyncHooks`.